packages/3d-web-client-core/src/rendering/composer.ts

```diff
@@ -91,11 +91,17 @@ export class Composer {
   private readonly gaussGrainPass: ShaderPass;
 
   private ambientLight: AmbientLight | null = null;
+  private environmentConfiguration: EnvironmentConfiguration;
 
   public sun: Sun | null = null;
   public spawnSun: boolean;
 
-  constructor(scene: Scene, camera: PerspectiveCamera, spawnSun: boolean = false) {
+  constructor(
+    scene: Scene,
+    camera: PerspectiveCamera,
+    spawnSun: boolean = false,
```

Comment:
I'd eliminate the default value for the `spawnSun` argument (explicit is always better) otherwise we'll need to make it the last constructor argument (wouldn't hurt much as we're changing the constructor signature anyway, but as we may want to instantiate the composer with just an empty `{}` to be an empty `environmentConfiguration` I think it would be good to have it as last argument.).

Reply:
Whoops, completely missed that, you're right. I think it makes sense to remove the default argument